tracer/test/Directory.Build.props

```diff
@@ -17,11 +17,29 @@
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
     <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
+    <PackageReference Include="DatadogTestLogger" Version="0.0.6">
+      <CopyToOutputDirectory>lib\$(TargetFramework)\*</CopyToOutputDirectory>
+      <FolderName>datadogtestlogger</FolderName>
+    </PackageReference>
   </ItemGroup>
 
   <!-- StyleCop -->
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)GlobalSuppressions.cs" Link="GlobalSuppressions.test.cs" />
   </ItemGroup>
 
+
+  <Target Name="PackageReferenceCopyToOutput" AfterTargets="Build">
+    <ItemGroup>
+      <PackageReferenceFiles Condition="%(PackageReference.CopyToOutputDirectory) != '' AND %(PackageReference.FolderName) != ''"
+        Include="$(NugetPackageRoot)\%(PackageReference.FolderName)\%(PackageReference.Version)\%(PackageReference.CopyToOutputDirectory)" />
+    </ItemGroup>
+    <Message Condition="%(PackageReference.CopyToOutputDirectory) != ''" 
+             Text="> $(NugetPackageRoot) | %(PackageReference.FolderName) | %(PackageReference.Version) | %(PackageReference.CopyToOutputDirectory) [$(NugetPackageRoot)\%(PackageReference.FileName)\%(PackageReference.Version)\%(PackageReference.CopyToOutputDirectory) | $(OutDir)]" Importance="high"/>
+    <Message Condition="%(PackageReference.CopyToOutputDirectory) != '' AND '@(PackageReferenceFiles->'%(Identity)')' != ''"
+             Text="  Copy @(PackageReferenceFiles->'%(Identity)') to $(OutDir)"
+             Importance="high"/>
+    <Copy SourceFiles="@(PackageReferenceFiles)" DestinationFolder="$(OutDir)" />
+  </Target>
```

Comment:
This mess is why we have Nuke to do it in C# 😛

Reply:
I know ...😬 but it works!